scripts/build_mod_ic.py

```diff
@@ -88,6 +88,17 @@
         "pin_shape": (1.5, 0.5),
     },
 
+    # SOP-16
+    "SOP-16": {
+        "rows": 2,
+        "pins": 16,
+        "pin_pitch": 1.27,
+        "row_pitch": 7.05,
+        "pad_shape": (2.9, 0.6),
+        "chip_shape": (5.6, 10.5),
+        "pin_shape": (1.3, 0.51),
+    },
+
```

Comment:
where did you find any reference for the footprint? I can only hunt down http://www.ti.com/lit/ml/msop002a/msop002a.pdf which doesn't have a footprint shape.